src/commands/development.ts

```diff
@@ -29,74 +29,107 @@ export default class Development extends Command {
+                    this.error("Service not supplied");
+
+                    return;
+                }
+
+                const servicesDisabled = services.map(service => {
+                    if (this.validateService(service)) {
+                        this.disableService(service);
+
+                        return true;
+                    }
+
+                    return false
+                }).reduce((prev, next) => prev || next);
+
+                if (servicesDisabled) {
+                    await this.config.runHook("generate-runnable-docker-compose", {});
+                }
+                break;
         }
     }
 
-    private printAvailableServices (): void {
+    private printAvailableServices(): void {
         this.log("Available services:");
         for (const service of this.inventory.services.filter(item => item.repository !== null && item.repository !== undefined)) {
             this.log(` - ${service.name} (${service.description})`);
         }
     }
 
-    private validateService (serviceName: string) {
-        if (serviceName === null || serviceName === undefined) {
+    private validateService(serviceName: string): boolean {
+        if (serviceName == "" || serviceName === null || serviceName === undefined) {
             this.error("Service must be provided");
+            return false;
         }
+
         const service = this.inventory.services.find(item => item.name === serviceName);
         if (service === null || service === undefined) {
             this.error(`Service "${serviceName}" is not defined in inventory`);
+            return false;
         }
         if (service?.repository === null || service?.repository === undefined) {
             this.error(`Service "${serviceName}" does not have repository defined`);
+            return false;
         }
+
+        return true;
     }
 
-    private enableService (serviceName: string) {
+    private enableService(serviceName: string) {
         this.stateManager.includeServiceInLiveUpdate(serviceName);
         this.log(`Service "${serviceName}" is enabled`);
     }
 
-    private disableService (serviceName: string) {
+    private disableService(serviceName: string) {
         this.stateManager.excludeServiceFromLiveUpdate(serviceName);
         this.log(`Service "${serviceName}" is disabled`);
     }
 
-    private async cloneServiceRepository (serviceName: string): Promise<void> {
+    private async cloneServiceRepository(serviceName: string): Promise<void> {
         const service = this.inventory.services.find(item => item.name === serviceName) as Service;
 
-        const localPath = join(this.config.root, "..", "repositories", service.name);
+        const localPath = join(process.cwd(), "repositories", service.name);
         if (!existsSync(localPath)) {
             cli.action.start(`Cloning ${service.repository!.branch || "default"} branch of ${service.repository!.url} repository to ${relative(this.config.root, localPath)} directory`);
```

Comment:
very minor, 
just wondering if relative to `this.config.root` tells more than 
- relative to `process.cwd()`
- or even not relative at all and just `localPath`

Reply:
Good spot 👍🏼 Corrected thanks!